### PR TITLE
Release v0.5.1

### PR DESCRIPTION
**Pull request by oshoval**
Features:
* install-cni, Add resources requests for initContainer
* dependencies: use containernetworking/plugins v0.9.1

Signed-off-by: Or Shoval <oshoval@redhat.com>

### CODE REVIEW

File: version/description [hunk 1]
```diff
@@ -1,9 +1,7 @@
...
+v0.5.1
 Features:
-* Link pods to priority classes #42
-* Rename "master" API #41 to use inclusive language
-
+* install-cni, Add resources requests for initContainer
+* dependencies: use containernetworking/plugins v0.9.1
 ```
-docker pull quay.io/kubevirt/macvtap-cni:v0.5.0
+docker pull HANDLER_IMAGE
```
Comment (oshoval) [inline version/description:6]: @maiqueb @phoracek 
what is the process please?
can you assist in creation of quay.io/kubevirt/macvtap-cni:v0.5.1 ?

Comment (maiqueb) [reply version/description:6]: You'll have to manually make this right, I guess that something is not correctly replacing the place holder with the actual name of the image. 

Update it manually so it features the img you're creating - i.e. `quay.io/kubevirt/macvtap-cni:v0.5.1` .

Comment (oshoval) [reply version/description:6]: ah ok and then just the CI will create it ?

Comment (maiqueb) [reply version/description:6]: yes.

Comment (oshoval) [reply version/description]: thanks, done

**Review by maiqueb — APPROVED** [review]

**Comment by RamLavi** [comment]
Amm.. Why are we issuing a patch release and not a minor one?

**Comment by oshoval** [comment]
> Amm.. Why are we issuing a patch release and not a minor one?

interesting, 
i see sometimes there is patch and sometimes there is minor
anyhow `make minor` i can't do myself because it also pushes to quay and i don't have credentials i think

@maiqueb @phoracek 
should we create a minor or patch ?

**Comment by maiqueb** [comment]
> > Amm.. Why are we issuing a patch release and not a minor one?
> 
> interesting,
> i see sometimes there is patch and sometimes there is minor

And sometimes there's major. 

> anyhow `make minor` i can't do myself because it also pushes to quay and i don't have credentials i think

? It would be the bot building images and pushing them to quay. You don't need credentials.
> 
> @maiqueb @phoracek
> should we create a minor or patch ?

Depends on the change; bumping the dependency is clearly material for a patch release; I guess it depends on what exactly is the change you pushed. Is that a new functionality ?... Is a bugfix ?... 

I personally don't think it is new functionality, but let's hear @RamLavi out.

**Comment by oshoval** [comment]
> Depends on the change; bumping the dependency is clearly material for a patch release; I guess it depends on what exactly is the change you pushed. Is that a new functionality ?... Is a bugfix ?...
> 
> I personally don't think it is new functionality, but let's hear @RamLavi out.

Indeed not new functionality, https://github.com/kubevirt/macvtap-cni/pull/48
just add minimum requests for the initContainer

**Comment by RamLavi** [comment]
> > Depends on the change; bumping the dependency is clearly material for a patch release; I guess it depends on what exactly is the change you pushed. Is that a new functionality ?... Is a bugfix ?...
> > I personally don't think it is new functionality, but let's hear @RamLavi out.
> 
> Indeed not new functionality imho, #48
> just add minimum requests for the initContainer

Following what you said I tend to agree, Let's keep this.

**Review by RamLavi — APPROVED** [review]
/lgtm

**Pull request merged**